core/domain/stats_services.py

```diff
@@ -122,6 +169,14 @@ def record_answers(
         submitted_answer_list):
     """Optimally record a group of answers using an already loaded exploration..
     The submitted_answer_list is a list of SubmittedAnswer domain objects.
+
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: int. The version of the exploration.
```

Comment:
I just did a quick grep and I did see instances of get_state_answers in admin.py, and get_sample_answers in the stats services test. So ... are we looking at different things, or is your grep not working?

In general, try to make use of two things: (a) data you glean from callsites, and also (b) reading the code, understanding what it does, and reasoning about what the allowed values are.